app/templates/newmaps.html

```diff
@@ -155,4 +252,4 @@ <h1 class="page-header" id="map-header">Philly Food Finder</h1>
 <!-- FAQ -->
 {% block content_faq %}
```

Comment:
Can you remove the `content_faq` block and corresponding end block? We are no longer using it.